packages/builder-worker/src/nodes/append-module.ts

```diff
@@ -236,6 +244,8 @@ export class FinishAppendModulesNode implements BuilderNode {
       this.bundle
     );
 
+    log(`  completed serializing bundle ${Date.now() - start}`);
```

Comment:
The explicit spaces to start these log messages seem like a smell. Maybe the `source` idea could help eliminate the need for theme?

Reply:
Good point. Totally agree